players/troubleshooting/crash-reports.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Crash Reports
+description: Learn what to do with crash reports, and how to read them.
+---
+
+# Crash Reports
+
+::: tip
+If you're having any difficulty with finding the cause of the crash, you can ask for help in the [Fabric Discord](https://discord.gg/v6v4pMv) in the `#player-support` or `#server-admin-support` channel.
+:::
+
+Crash reports are a very important part of troubleshooting issues with your game or server. They contain a lot of information about the crash, and can help you find the cause of the crash.
+
+## Finding Crash Reports
+
+Crash reports are stored in the `crash-reports` folder in your game directory. If you are using a server, they are stored in the `crash-reports` folder in the server directory.
+
+For third party launchers, you should refer to their documentation on where to find crash reports.
+
+Crash reports can be found in the following locations:
+
+::: code-group
+
+```:no-line-numbers [Windows]
+%appdata%\.minecraft\crash-reports
+```
+
+```:no-line-numbers [macOS]
+~/Library/Application Support/minecraft/crash-reports
+```
+
+```:no-line-numbers [Linux]
+~/.minecraft/crash-reports
+```
+
+:::
+
+## Reading Crash Reports
+
+Crash reports are very long, and can be very confusing to read. However, they contain a lot of information about the crash, and can help you find the cause of the crash.
+
+For this guide, we will be using the [following crash report as an example.](https://gist.github.com/IMB11/8fb305a98212bcad2d60f8bd4b041f76)
```

Comment:
Should ideally be hosted here.

Reply:
There's not a very good way to include the log file on the page without it taking up 80% of the content, I'll investigate some solutions.